### PR TITLE
Viite 360 roadaddress segments are in wrong order after moving

### PR DESCRIPTION
**Pull request by Campanha**

### CODE REVIEW

**Comment by tvenhola** [comment]
Looks good to me

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala [hunk 1]
```diff
@@ -454,4 +430,274 @@ class RoadAddressServiceSpec extends FunSuite with Matchers{
...
+    val replacement1s = addressLinks.filter(l => l.linkId == 1717639 || l.linkId==499897217)
+    val replacement1t = addressLinks.filter(l => l.linkId == 500130192)
+    replacement1s.size should be (2)
+    replacement1t.size should be (1)
+    val result1 = roadAddressService.transferRoadAddress(replacement1s, replacement1t, User(0L, "foo", Configuration()))
+    sanityCheck(result1)
+    result1.head.startMValue should be (0.0)
+    result1.head.startAddressM should be (replacement1s.map(_.startAddressM).min)
+    result1.head.endAddressM should be (replacement1s.map(_.endAddressM).max)
+    //    replacement1s.foreach(l=>println(prettyPrint(l)))
```
Comment (pedrosag) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala]: Is the code in 454, 455 and 456 lines needed or can it be deleted?

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala [hunk 2]
```diff
@@ -454,4 +430,274 @@ class RoadAddressServiceSpec extends FunSuite with Matchers{
...
+    val replacement2t = addressLinks.filter(l => l.linkId == 500130201)
+    replacement2s.size should be (2)
+    replacement2t.size should be (1)
+    val result2 = roadAddressService.transferRoadAddress(replacement2s, replacement2t, User(0L, "foo", Configuration()))
+    sanityCheck(result2)
+
+    result2.head.startMValue should be (0.0)
+    result2.head.startAddressM should be (replacement2s.map(_.startAddressM).min)
+    result2.head.endAddressM should be (replacement2s.map(_.endAddressM).max)
+    //    replacement2s.foreach(l=>println(prettyPrint(l)))
```
Comment (pedrosag) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala]: Is the code in 468, 469 and 470 lines needed or can it be deleted?

Comment (tvenhola) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala]: let's clean that debugging code

**Comment by tvenhola** [comment]
Merged, pedrosag at lunch but review items corrected.

**Pull request merged**